src/views/Create.vue

```diff
@@ -81,7 +81,9 @@ const settingsForm: SpaceSettings = reactive(
     minVotingDuration: 0,
     maxVotingDuration: 86400,
     proposalThreshold: '1',
-    quorum: '1'
+    ...(!evmNetworks.includes(selectedNetworkId.value) && {
+      quorum: '1'
+    })
```

Comment:
This doesn't really work anyway (https://github.com/snapshot-labs/sx-ui/pull/474/files#r1133918323), so let's revert it.